src/Common/src/TypeSystem/Common/TypeHashingAlgorithms.cs

```diff
@@ -138,11 +138,28 @@ public static int ComputeGenericInstanceHashCode<ARG>(int genericDefinitionHashC
         /// <param name="typeHashcode">Hashcode of the type that owns the method</param>
         /// <param name="nameOrNameAndGenericArgumentsHashcode">Hashcode of either the name of the method (for non-generic methods) or the GenericInstanceHashCode of the name+generic arguments of the method.</param>
         /// <returns></returns>
-        public static int ComputeMethodHashcode(int typeHashcode, int nameOrNameAndGenericArgumentsHashcode)
+        public static int ComputeMethodHashCode(int typeHashcode, int nameOrNameAndGenericArgumentsHashcode)
```

Comment:
Fix names of arguments and locals in this file to use the capital `C` casing as well?